CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ This is an example of a brief overview of the _Major_ or _Minor_ version changes
 
 ---
 
+## [9.10] Toast performance
```

Comment:
technically this is a patch version, but I wanted to give us a minor version milestone in case I broke something - it may or may not be a good idea to make an exception for this

Reply:
hmmm 🙂